featurebyte/api/api_object.py

```diff
@@ -720,7 +720,9 @@ def info(self, verbose: bool = False) -> Dict[str, Any]:
         client = Configurations().get_client()
         response = client.get(url=f"{self._route}/{self.id}/info", params={"verbose": verbose})
         if response.status_code == HTTPStatus.OK:
-            return PrettyDict(response.json())
+            info = response.json()
+            info["class_name"] = self.__class__.__name__
+            return InfoDict(info)
         raise RecordRetrievalException(response, "Failed to retrieve object info.")
```

Comment:
Currently, we use `info` in `__repr__`:

```
    def __repr__(self) -> str:
        info_repr = ""
        try:
            info_repr = repr(self.info())
        except (RecordCreationException, RecordRetrievalException):
            # object has not been saved yet
            pass

        return construct_repr_string(self, info_repr)
```

Since `repr()` is used, `_repr_html_` will not be triggered. That means when displaying api object, it will show dictionary. Just want to check whether this is intended.

Reply:
Indeed. I will implement `_repr_html_` function in `ApiObject` class so that we can get the html representation for api objects as well in a notebook environment.